rails_app/app/jobs/deals_scrape_job.rb

```diff
@@ -11,33 +11,39 @@ def perform(*args)
     deals_to_parse.each do |deal|
       # new_deal_status = Settings.parser_helpers.join_parsers[deal.provider].constantize.new(deal.url)
       begin
-        if deal.updated_at.nil? or deal.updated_at + deal.scrape_interval < Time.now
-          new_deal = BasicParser.new(deal.url).parse
-          deal_identifier = Digest::MD5.hexdigest(deal.url)
-          @@graphite.send_raw({
-            "deals.#{deal_identifier}.price" => new_deal.price,
-            "deals.#{deal_identifier}.time_since_update" => (Time.now - deal.updated_at),
-            "deals.#{deal_identifier}.smart_interval" => deal.scrape_interval,
-          })
-          deal.update!(price: new_deal.price, updated_at: Time.now)
-          processed_deals = processed_deals + 1
-        end
+        new_deal = BasicParser.new(deal.url).parse
+        deal_identifier = Digest::MD5.hexdigest(deal.url)
```

Comment:
I think we can change it to CamelCase from `deal.name`

Reply:
am i blind or there is no deal.name in this file?